tools/universe/package_builder.py

```diff
@@ -16,7 +16,7 @@
 logger = logging.getLogger(__name__)
 logging.basicConfig(level=logging.DEBUG, format="%(message)s")
 
-_jre_url = "https://downloads.mesosphere.com/java/server-jre-8u192-linux-x64.tar.gz"
+_jre_url = "https://downloads.mesosphere.com/java/OpenJDK8U-jre_x64_linux_hotspot_8u212b03.tar.gz"
```

Comment:
Let's try to follow a similar pattern here: https://github.com/mesosphere/dcos-commons/pull/3040/files#diff-ad95ea0f28b517acb42597b92e757a1aR19